.yamato/_abv.yml

```diff
@@ -14,11 +14,8 @@ smoke_test_Standalone_trunk:
     commands:
       -  curl -s https://artifactory.internal.unity3d.com/core-automation/tools/utr-standalone/utr.bat --output TestProjects/SRP_SmokeTest/utr.bat
       -  pip install unity-downloader-cli --index-url https://artifactory.prd.it.unity3d.com/artifactory/api/pypi/pypi/simple --upgrade
-      -  cd TestProjects/SRP_SmokeTest && unity-downloader-cli --source-file ../../unity_revision.txt -c editor --wait --published-only
+      -  cd TestProjects/SRP_SmokeTest && unity-downloader-cli -u 4ae7c5e87f33525aa3c51e804b161e43d1c016d6 -c editor --wait --published-only
```

Comment:
I am wondering if we can continue to retrieve the unity revision from an external file instead of having it hardcoded in the yaml. We would have to specify the file at this line (like before) and the `editor-pinning-merge-to-target` would commit this specific file only. 
This way there is no diff at this line when a dev wants to merge their branch with potential changes to this file into master (which potentially has a different editor version since the developer branched off of it). There is only a diff with the specific file.

Reply:
I like this idea! Definitely in for having less changes per these automated commits, but I am wondering how to do this since I'm not sure we can do it as before: before, editor-priming called the unity-downloader-cli to get the revision, and stored it in a file (i.e. this file is not in the repo, it gets created on the agent as Yamato runs). In current case (unless we create a job similar to priming), I think we would need to keep these files in the repo and have a file for each platform (i.e. `unity_revision_win.txt`, ...), where each of the files would essentially be a string extracted from the `_latest_editor_versions.metafile`. This could be done e.g. by a) another script which extracts latest_internal per each platform, and stores each of these in a separate file, or b) let editor pinning store versions straightaway in these `unity_revision_[platform].txt` files. Both of these options have their good and bad, but this could reduce to nr of changes being proportional to nr of platforms, instead of nr of ymls! Or perhaps there is another approach I didn't think of yet

Reply:
taking this thread to slack!